### PR TITLE
Copy authors and maintainers info from module

### PR DESCRIPTION
**Pull request by webknjaz**
Fixes ansible/ansibullbot#1049

cc @jctanner @gundalow

### CODE REVIEW

File: tests/unit/utils/test_component_tools.py [hunk 1]
```diff
@@ -103,8 +110,9 @@ def test_get_meta_for_file_powershell(self):
             }
         }
         result = self.component_matcher.get_meta_for_file(u'lib/ansible/modules/windows/win_ping.ps1')
-        self.assertEqual(result[u'labels'], [u'windoez'])
-        self.assertEqual(result[u'maintainers'], [u'jborean93'])
+        assert result[u'labels'] == [u'windoez']
+        expected_maintainers = sorted([u'cchurch', u'jborean93'])
+        assert sorted(result[u'maintainers']) == expected_maintainers
```
Comment (gundalow) [inline tests/unit/utils/test_component_tools.py]: Is it worth adding a test of `ignore` in here as well?

**Comment by gundalow** [comment]
Tanner said
> Testing ignore seems valid

File: tests/unit/utils/test_component_tools.py (hunk 1 above)
Comment (webknjaz) [reply tests/unit/utils/test_component_tools.py]: ignore: done
no authors: impossible
authors + botmeta: dunno how it's different from what we have now

File: tests/unit/utils/test_component_tools.py [hunk 2]
```diff
@@ -86,12 +86,19 @@ def test_get_meta_for_file_pyfile(self):
         self.component_matcher.file_indexer.botmeta = self.component_matcher.BOTMETA = {
             u'files': {
                 u'lib/ansible/modules/packaging/os/yum.py': {
+                    u'ignored': [u'verm666'],
```
Comment (webknjaz) [inline tests/unit/utils/test_component_tools.py]: @gundalow ^

**Pull request merged**